src/moveFile.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+function moveFile(fileName, moveTo) {
+  let movePath = path.join(__dirname, '..', moveTo);
+
+  movePath = movePath.endsWith('/') ? movePath.slice(0, -1) : movePath;
+
+  const isDir = fs.existsSync(movePath);
+  const filePath = path.join(__dirname, fileName);
+  const data = fs.readFileSync(filePath, 'utf-8');
+
+  if (!isDir) {
+    throw new Error(`Directory ${movePath} does not exist`);
+  }
+
+  movePath = path.join(movePath, fileName);
+
+  fs.writeFileSync(movePath, data);
+  fs.rmSync(filePath);
```

Comment:
You are using rmSync which can block the entire thread until the operation finishes. Instead, consider using the asynchronous version rm and handle the file content in its callback.